src/main.py

```diff
@@ -15,6 +15,7 @@
     orientation: 'vertical'
     ZBarCam:
         id: zbarcam
+        index: 1    # or try a different value
```

Comment:
How about keeping it to `0` since it's a runnable demo and `0` would work for more cases.
Also mentioning it's optional, e.g.
```suggestion
        # optional, override the camera index (default 0)
        camera_index: 0    # or try a different value
```